src/Lykke.Job.Bitcoin/Functions/UpdateBalanceFunctions.cs

```diff
@@ -59,6 +65,8 @@ public async Task UpdateBalances()
 
         private async Task ProcessBlock(int height)
         {
+            _log.Info("Processing block", context: height);
```

Comment:
better to make:

```suggestion
            _log.Info("Processing block", context: new { Height = height });
```

This will help to search logs in kibana